package.json

```diff
@@ -33,7 +33,8 @@
     "moment": "^2.10.6",
     "react-native-button": "^1.3.1",
     "react-native-gifted-spinner": "0.0.3",
-    "react-native-parsed-text": "0.0.11"
+    "react-native-parsed-text": "0.0.11",
+    "react-native-autogrow-textinput": "github:faceyspacey/react-native-autogrow-textinput"
```

Comment:
I'm not sure about this. There isn't any added security for this; can you register this on NPM? If you decide for some reason to change/delete the repo, it's gone!